themes/xresources/assets.lua

```diff
@@ -48,9 +48,18 @@ end
 
 
 function theme_assets.wallpaper(bg, fg, alt_fg, s)
-    s = s or screen.primary
-    local height = s.workarea.height
-    local width = s.workarea.width
```

Comment:
By the way, why is this using the workarea instead of the geometry of the screen? That seems wrong to me.